src/hooks/useCustomToast.ts

```diff
@@ -0,0 +1,41 @@
+import { useToast } from 'react-native-toast-notifications';
+
+import { ToastTypes } from '@/components/common/Toast';
+
+function useCustomToast() {
+  const toast = useToast();
+
+  const showSuccess = (title: string, message: string) => {
```

Comment:
we could wrap them in useCallback, what u think?

Reply:
Yes we could!